modules/@apostrophecms/page/index.js

```diff
@@ -1276,11 +1276,11 @@ database.`);
             if (parent._id !== oldParent._id) {
               const matchOldParentSlugPrefix = new RegExp('^' + self.apos.util.regExpQuote(self.apos.util.addSlashIfNeeded(oldParent.slug)));
               if (moved.slug.match(matchOldParentSlugPrefix)) {
-                let slugStem = parent.slug;
-                if (slugStem !== '/') {
-                  slugStem += '/';
-                }
-                moved.slug = moved.slug.replace(matchOldParentSlugPrefix, self.apos.util.addSlashIfNeeded(parent.slug));
+                const relativeSlug = path.relative(parent.slug, moved.slug);
```

Comment:
Not suitable for page slugs because on some platforms it may return a platform-dependent path separator.

Reply:
I know right now our documentation says native Windows is not supported, but it's not best practice to use `path` contrary to its intended purpose, and the way things are going with feedback from customers I think we will have to ease up and support native Node.js on Windows soon so let's not dig the hole deeper on that